libs/store/src/lib/loggers/tournament.ts

```diff
@@ -0,0 +1,16 @@
+import { AppErrorCode, AppMessageLogType } from '@razor/models';
+import { Dispatch } from '../store';
+
+export const tournamentNotFound = async (
+  dispatch: Dispatch,
+  tid: string,
+  additionalMessage?: string,
+) => {
+  dispatch.game.sendLogMessage({
+    message: `Tournament with id ${tid} does not exist`,
+    code: AppErrorCode.TournamentNotExists,
+    related: additionalMessage ? additionalMessage : '',
+    type: AppMessageLogType.Error,
+  });
+  return;
```

Comment:
This statement is redundant.

Reply:
Fixed in PR #14 under commit a30c31abb7742dfe160b204a5bca3285d6a0d65a